container.go

```diff
@@ -715,3 +719,21 @@ func (c *Container) addDrive(create bool) (err error) {
 
 	return nil
 }
+
+func (c *Container) removeDrive() (err error) {
+	if c.state.Fstype != "" {
```

Comment:
Once again, a boolean with a different name (something like `pluggedDrive` maybe) would be much clearer to understand the code here.

Reply:
A value in Fstype means that drive is plugged, I was trying to avoid adding an extra state variable. Probably a container function isDrivePugged() which checks for fstype and using that function instead would make the code clearer. What do you think?

Reply:
Yes good idea. This function would hide the "complexity" and would help making things clearer.